config/env/production.js

```diff
@@ -78,6 +78,13 @@ module.exports = {
     callbackURL: '/api/auth/paypal/callback',
     sandbox: false
   },
+  owasp: {
+      allowPassphrases       : true,
+      maxLength              : 128,
+      minLength              : 10,
+      minPhraseLength        : 20,
+      minOptionalTestsToPass : 4,
+  },
```

Comment:
If the config is in default and someone changes it there (as opposed to duplicating it and changing it in the local), the tests might fail. Say you reduce the minlength requirement, that test would fail... although that may not be a bad thing.